lib/models/channel_account.ex

```diff
@@ -4,11 +4,12 @@ defmodule ExMicrosoftBot.Models.ChannelAccount do
   """
 
   @derive [Poison.Encoder]
-  defstruct [:id, :name]
+  defstruct [:id, :name, :aadObjectId]
```

Comment:
There are some test cases without this field:

https://github.com/PagerDuty/ex_microsoftbot/blob/1873cb753ef6435ee4a22cd5eebb605215ef6faa/test/models/channel_acount_test.exs#L8

If the change is not temporary, then we should cover `aadObjectId` with tests.

Reply:
Good point. Thanks!
The tests have been updated.